File: app/src/main/java/org/growingstems/scouting/MatchFragment.java
package org.growingstems.scouting;


import androidx.fragment.app.Fragment;

import org.frc836.database.MatchStatsStruct;

public abstract class MatchFragment extends Fragment {
    public abstract void saveData(MatchStatsStruct data);
    public abstract void loadData(MatchStatsStruct data);
}

File: app/src/main/java/org/growingstems/scouting/MatchFragment.kt
package org.growingstems.scouting

import androidx.fragment.app.Fragment
import org.frc836.database.MatchStatsStruct

abstract class MatchFragment : Fragment() {
    abstract fun saveData(data: MatchStatsStruct)
    abstract fun loadData(data: MatchStatsStruct)
}
